src/main/java/teammates/common/datatransfer/questions/FeedbackMcqQuestionDetails.java

```diff
@@ -489,6 +489,11 @@ public String getQuestionTypeChoiceOption() {
         return null;
     }
 
+    @Override
+    public boolean isStudentCommentsOnResponsesAllowed() {
+        return true;
```

Comment:
Just a clarification, this is not done yet right?

Reply:
This is done. Students can add comments in MCQ type questions.

Reply:
I meant whether this method is done? Since it is, why don't you just have a boolean variable instead of having a function to just return a single value? Similar for the other methods.

Reply:
Because I thought right now isStudentCommentsOnResponsesAllowed() is fixed, so it shouldn't be used included as data member in class.
Also instructor comments has been implemented like this.
Maybe when we implement the feature to toggle student comments, we can change it to variable.